lib/timeline/component/ItemSet.js

```diff
@@ -356,16 +357,17 @@ ItemSet.prototype.setOptions = function(options) {
 
     if ('editable' in options) {
       if (typeof options.editable === 'boolean') {
-        this.options.editable.updateTime  = options.editable;
```

Comment:
:angry: That's why I don't like these code-indent-formatting-thinks. (I hope we get rid of them in the future). 
@simo9000 No problem here, just a comment...